library/src/main/java/com/anjlab/android/iab/v3/BillingBase.java

```diff
@@ -28,6 +28,9 @@
 		this.context = context;
 	}
 
+	public BillingBase() {
```

Comment:
can you explain the purpose of having public constructor here?

Reply:
@serggl Since i added a constructor(requiring context) in BillingProcessor for the connect(this) method, the BillingBase needed a public constructor since it extends BillingProcessor